src/docker-osx-dev

```diff
@@ -562,7 +554,7 @@ function install_dependencies {
   brew update
 
   brew_install "caskroom/cask/brew-cask" "Cask" "" false
-  brew_install "virtualbox" "VirtualBox" "VBoxManage" true
+  brew_install "virtualbox" "VirtualBox" true
```

Comment:
The whole virtualbox line is not needed. As boot2docker has the dependency anways.